documentation/ag-grid-docs/src/components/reference-documentation/components/Property.tsx

```diff
@@ -196,141 +196,168 @@ export const Property: FunctionComponent<{
+                                                        aria-label={`See more details about ${more?.name}`}
+                                                    >
+                                                        <Icon
+                                                            className={`${styles.chevron} ${isExpanded ? 'expandedIcon' : ''}`}
+                                                            name="chevronDown"
+                                                        />
+                                                    </button>
+                                                )}
+                                                <span className={styles.metaValue}>{propertyType}</span>
+                                                <div className={styles.actions}></div>
+                                            </div>
+                                        )}
 
-                    <div className={styles.metaList}>
-                        <div
-                            title={typeUrl && isObject ? getInterfaceName(name) : propertyType}
-                            className={styles.metaItem}
-                            onClick={() => setExpanded(!isExpanded)}
-                        >
-                            <span className={styles.metaLabel}>Type</span>
-                            {typeUrl ? (
-                                <a
-                                    className={styles.metaValue}
-                                    href={typeUrl}
-                                    target={typeUrl.startsWith('http') ? '_blank' : '_self'}
-                                    rel="noreferrer"
-                                >
-                                    {isObject ? getInterfaceName(name) : propertyType}
-                                </a>
-                            ) : (
-                                <span className={styles.metaValue}>{propertyType}</span>
-                            )}
+                                        {formattedDefaultValue != null && (
+                                            <div className={styles.metaItem}>
+                                                <span className={classnames(styles.metaValue, styles.defaultValue)}>
+                                                    <span>default: </span>
+                                                    {formattedDefaultValue}
+                                                </span>
+                                            </div>
+                                        )}
+
+                                        {isInitial && (
+                                            <div className={classnames(styles.metaItem, styles.initialItem)}>
+                                                <a
+                                                    className={styles.initialLabel}
+                                                    href="https://ag-grid.com/react-data-grid/grid-interface/#initial-grid-options"
```

Comment:
I think you want this link to be relative to the deployment:

```
import { urlWithPrefix } from '@utils/urlWithPrefix';
```

```suggestion
                                                    href={urlWithPrefix({ url: "./grid-interface/#initial-grid-options", framework })}
```